sdk/metric/selector/simple/simple.go

```diff
@@ -65,15 +65,18 @@ func lastValueAggs(aggPtrs []*aggregator.Aggregator) {
 	}
 }
 
+func histogramAggs(aggPtrs []*aggregator.Aggregator,
+	desc *sdkapi.Descriptor, opts ...histogram.Option) {
+	aggs := histogram.New(len(aggPtrs), desc, opts...)
+	for i := range aggPtrs {
+		*aggPtrs[i] = &aggs[i]
+	}
+}
+
 func (selectorInexpensive) AggregatorFor(descriptor *sdkapi.Descriptor, aggPtrs ...*aggregator.Aggregator) {
 	switch descriptor.InstrumentKind() {
 	case sdkapi.GaugeObserverInstrumentKind:
 		lastValueAggs(aggPtrs)
-	case sdkapi.HistogramInstrumentKind:
```

Comment:
Just commenting here because it took me a few seconds to understand why this code was removed.
The `sumAggs` method does the same thing, so there's no need for the repetition.